src/Dolo.jl

```diff
@@ -35,17 +30,26 @@ export yaml_import, eval_with, evaluate, evaluate!, model_type, name, filename,
 
 export time_iteration, value_iteration, steady_state_residuals, simulation
 
-# set up core types
-abstract AbstractModel{ID}
-abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
-abstract AbstractNumericModel{ID} <: AbstractModel{ID}
+export ModelCalibration, FlatCalibration, GroupedCalibration
 
-typealias ASM AbstractSymbolicModel
-typealias ANM AbstractNumericModel
+# set up core types
+abstract AbstractSymbolicModel{ID}
+# abstract AbstractSymbolicModel{ID} <: AbstractModel{ID}
+abstract AbstractModel{ID} <: AbstractSymbolicModel{ID}
```

Comment:
Now we don't have a NumericalModel type. Just Model. So AbstractNumericModel->AbstractModel.
SymbolicModel is just a thin layer to that data. We could call it smthg elsr